test/group.test.js

```diff
@@ -105,6 +105,21 @@ describe('Group Methods', () => {
     })
   })
 
+  it('searchGroups() returns groups that match the query', () => {
+    return searchGroups('noblox.js').then((res) => {
```

Comment:
Follow-up regarding my limit concern; won't this test run far longer than it needs to?